draft-ietf-mops-streaming-opcons.md

```diff
@@ -219,6 +288,46 @@ Presentations:
+
+Although TCP protocol behavior has changed over time, the common practice of implementing TCP as part of an operating system kernel has acted to limit how quickly TCP behavior can change, even with the widespread use of automated operating system update installation on many end-user systems, so that streaming media providers could have a reasonable expectation that they could understand TCP transport protocol behaviors, and that those behaviors would remain relatively stable in the short term. 
+
+## The QUIC Protocol and QUIC Protocol Behavior
+
+The QUIC protocol {{I-D.ietf-quic-transport}}, developed from a proprietary protocol into an IETF standards-track protocol {{I-D.ietf-quic-transport}}, turns many of the statements made in {{udp-behavior}} and {{tcp-behavior}} on their heads. 
+
+The standardized QUIC protocol is used to carry HTTP traffic, but instead of being TCP-based, HTTP/3 {{I-D.ietf-quic-http}} is encapsulated in QUIC, which is then encapsulated in UDP, so streaming operators (and network operators) will see traffic that looks like HTTP, but is carried over UDP (specifically, using UDP port 443 {{Port443}}). Network operators may already be blocking UDP traffic on this port, since HTTP over TLS/SSL has used TCP, not UDP. Even if UDP traffic using this port is not blocked, traffic using this port may be severely rate-limited, since HTTP/3 over QUIC has the potential to send much more traffic over UDP than the network operator expects. 
+
+As noted elsewhere in this document, the QUIC protocol encrypts almost all of its transport parameters, and all of its payload, so any intermediaries that network operators may be using to perform analytics or even participate in current HTTP conversations will not work for HTTP/3 without making changes to their networks.
+
+As noted in {{tcp-behavior}}, there is increasing interest in transport protocol behaviors that responds to delay measurements, instead of responding to packet loss. These behaviors may deliver improved user experience, but in some cases have not responded to sustained packet loss, which exhausts available buffers along the end-to-end path that may affect other users sharing that path. The standardized QUIC protocol includes a congestion-control mechanism {{I-D.ietf-quic-recovery}} that is loss-based, and is intended to have roughly the behavior of TCP NewReno {{RFC6582}}, but the signals QUIC provides for congestion control are intended to be generic, and a sender can unilaterally chose a different algorithm to use, which might be loss-based, like CUBIC {{RFC8312}}, a delay-based congestion controller like COPA or BBR, or something completely different. 
```

Comment:
I would invert the sentences about QUIC a a little and say:

QUIC {{I-D.ietf-quic-transport}} provides a set of congestion control hooks that can be use for algorithm agility. {{I-D.ietf-quic-recovery}} defines a basic algorithm that behaves roughly similar TCP NewReno {{RFC6582}}. However, QUIC senders can and do unilaterally chose to use different algorithms such as loss-based CUBIC {{RFC8312}}, delay-based COPA or BBR, or even something completely different

Reply:
Thanks for this suggestion. I used very close to your text (with minor dorking).